src/index.js

```diff
@@ -1,41 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
 import {
   BrowserRouter as Router,
   Route,
   useHistory,
   Switch,
 } from 'react-router-dom';
 import { Security, LoginCallback, SecureRoute } from '@okta/okta-react';
-
 import 'antd/dist/antd.less';
-
 import { NotFoundPage } from './components/pages/NotFound';
-import { TopicCreation } from './components/pages/TopicCreation';
-// import { ExampleListPage } from './components/pages/ExampleList';
-import { ProfileListPage } from './components/pages/ProfileList';
 import { LoginPage } from './components/pages/Login';
 import { HomePage } from './components/pages/Home';
-import { ExampleDataViz } from './components/pages/ExampleDataViz';
 import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
-
 import './styles/index.less';
-
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
 import thunk from 'redux-thunk';
 import logger from 'redux-logger';
 import { apolloReducer } from './state/reducers/apolloReducer';
+
+// create the global redux state
 const store = createStore(apolloReducer, applyMiddleware(thunk, logger));
 
+export { store };
```

Comment:
exporting the store hear is causing some tests to not pass.  This can be solved by putting the store in it's own file and default exporting from there.  See this for reference: [https://nono.ma/solved-invariant-violation-target-container-is-not-a-dom-element](url)